src/Context/BasicContext.php

```diff
@@ -929,11 +929,11 @@ public function iSeeTextInRegion($negate, $text, $region)
         Assert::assertNotNull($regionObj);
 
         $actual = $regionObj->getText();
-        $actual = preg_replace('/\s+/u', ' ', $actual);
-        $regex  = '/' . preg_quote($text, '/') . '/ui';
+        $actual = preg_replace('/\s+/u', ' ', $actual ?? '');
+        $regex  = '/' . preg_quote($text ?? '', '/') . '/ui';
 
-        if (trim($negate)) {
-            if (preg_match($regex, $actual)) {
+        if (trim($negate ?? '')) {
+            if (preg_match($regex ?? '', $actual ?? '')) {
```

Comment:
```suggestion
            if (preg_match($regex ?? '//', $actual ?? '')) {
```

Reply:
This was discussed in another PR - the warning emitted by the empty string is correct and we would be hiding an actual issue by using an empty regex. The intention here is just to avoid deprecation warnings.